packages/oas/oas-github-ci/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@medusajs/oas-github-ci",
+  "version": "1.0.0",
+  "description": "OAS Github CI",
+  "main": "scripts/build-openapi.js",
+  "files": [
+    "scripts"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/medusajs/medusa",
+    "directory": "packages/oas/oas-github-ci"
+  },
+  "publishConfig": {
+    "access": "restricted"
+  },
```

Comment:
question: Is there a reason for making this restricted? This setting is blocking us from publishing it in our pipeline

Reply:
The package is only relevant in our monorepo context. It doesn't provide any value as a standalone package. That's why I kept it as restricted. I don't think it needs to be published on npm.

Reply:
Makes sense. I think we'll have to change it to `private: true`. Changeset does not recognize this config when running the script to publish packages. If okay with you, I'll open a PR with it?

Reply:
Ah jeez, my bad for using the wrong flag in the package.json. Thanks for the PR.